frontend/src/pages/MapViewPage/MapViewPage.scss

```diff
@@ -1,6 +1 @@
 @import url("../../styles/common.scss");
-@import "leaflet/dist/leaflet.css";

```

Comment:
Why is this removed?

Reply:
This is double code, which is already in map component's scss file. When I refactored map into components, we talked about it.

Reply:
Ok, I forgot about that :D